climada/util/value_representation.py

```diff
@@ -14,16 +14,19 @@
 
 LOGGER = logging.getLogger(__name__)
 
-ABBREV = {1:'',
-          1000: 'K',
-          1000000: 'M',
-          1000000000: 'Bn',
-          1000000000000: 'Tn'}
+ABBREV = {
+    1:'',
+    1000: 'K',
+    1000000: 'M',
+    1000000000: 'Bn',
+    1000000000000: 'Tn'
+    }
 
 
 def sig_dig(x, n_sig_dig = 16):
     """
     Rounds x to n_sig_dig number of significant digits.
+    0, inf, Nan are returned unchanged.
     Examples: 1.234567 -> 1.2346, 123456.89 -> 123460.0
```

Comment:
Is this true? Why would it acutally correctly round up 56 to 60 in the 2nd example, but wrongly round down 6.7e-5 to 6e-5 in the first example?

Reply:
I do not understand... This rounds your number to a certain number of significant digits. In the example, `n_sig_dig=5` (I added this for clarity). Basically,  it counts from left up to 5 digits, then rounds that last digit and sets the remaining values to the right to 0.